### PR TITLE
Changelog for 0809 release

### PR DESCRIPTION
**Pull request by arkalyanms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,17 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Use hint diagnostics in vscode for better rendering in the editor (PR: [#69403](https://github.com/dotnet/roslyn/pull/69403))
```
Comment (jasonmalinowski) [inline CHANGELOG.md:12]:
```suggestion
* Change how hint diagnostics are shown so they aren't shown as blue squiggles (PR: [#69403](https://github.com/dotnet/roslyn/pull/69403))
```

Comment (jasonmalinowski) [reply CHANGELOG.md:12]: (that way folks understand the "blue squiggle" problem is addressed and they can potentially revert workarounds)

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,17 @@
...
+* Tell the language server to open projects if no solution exists (PR: [#6062](https://github.com/dotnet/vscode-csharp/pull/6062))
```
Comment (jasonmalinowski) [inline CHANGELOG.md:13]:
```suggestion
* Load projects even if there is no solution file (PR: [#6062](https://github.com/dotnet/vscode-csharp/pull/6062))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,17 @@
...
+* Show toast when project loading fails (PR: [#6060](https://github.com/dotnet/vscode-csharp/pull/6060))
+* Fix misleading lsp server logs (PR: [#69378](https://github.com/dotnet/roslyn/pull/69378))
+* Add support for code lens enable/disable options to roslyn LSP (PR: [#6001](https://github.com/dotnet/vscode-csharp/pull/6001))
+* Onboard Localization pipeline (PR: [#5990](https://github.com/dotnet/vscode-csharp/pull/5990))
+* Enable loading translated strings from razor TS code (PR: [#5962](https://github.com/dotnet/vscode-csharp/pull/5962))
+* Update typescript and eslint plugin to remove build warning (PR: [6002](https://github.com/dotnet/vscode-csharp/pull/6002))
```
Comment (jasonmalinowski) [inline CHANGELOG.md:19]: Should we include things that only impact us in changelogs like this? Or put them in a different section?

**Review by dibarbet — APPROVED** [review]

**Review by jasonmalinowski — APPROVED** [review]
A few suggestions to make the titles more user-facing rather than developer-facing.

File: CHANGELOG.md (hunk 3 above)
Comment (arkalyanms) [reply CHANGELOG.md:19]: My intent with including them was to give a rough idea of other work that were added to the extension, even though not user facing. But can move it to a separate section if thats better.

**Pull request merged**